doc/src/lexer_tutorial/002_raw_delimited_content.md

```diff
@@ -0,0 +1,347 @@
+# Lexing raw delimited content
+
+Our calculator example operated on numbers and arithmetic operators.
+There is no overlap between the characters for numeric digits (`0`, `1`, ...)
+the characters representing operators (`+`, `-`, ...) and parentheses
+(`(`, `)`), so it was easy to embed those tokens directly in the grammar,
+as we saw in the earlier sections.
+
+However, some languages do not have such easy separations at the lexical level.
```

Comment:
```suggestion
However, some languages do have overlapping lexical token.
```

Reply:
Addressed in fb568a3be286438944569aae7d25b33e6e0fa17b